microcosm_elasticsearch/tests/fixtures.py

```diff
@@ -0,0 +1,41 @@
+"""
+Test fixtures.
+
+"""
+from elasticsearch_dsl import Q, Text
+from microcosm.api import binding
+
+from microcosm_elasticsearch.indexing import make_index
+from microcosm_elasticsearch.models import Model
+from microcosm_elasticsearch.store import Store
+
+
+@binding("example_index")
```

Comment:
We always need to choose an index.

I'm assuming a microservice will have a few, static indexes right now, but we may end up with a broader index management layer eventually. In this case, we'll need some sort of delegation layer that lets a store find its current/correct index.